src/util/Util.js

```diff
@@ -741,6 +741,16 @@ class Util extends null {
       emoji_name: defaultReaction.name,
     };
   }
+
+  /**
+   * Calculates the default avatar index for a given user id.
+   * @param {Snowflake} userId - The user id to calculate the default avatar index for
+   * @returns {number}
+   * @ignore
```

Comment:
```suggestion
```

in original pr this method isnt igonerd

Reply:
@jaw0r3k I think it should be ignored because we don't need this function in the documentation :/

Reply:
users may want to use it

Reply:
hmm...oke